ui/app/AppLayouts/Chat/ChatColumn/ChatInput.qml

```diff
@@ -20,6 +20,13 @@ Rectangle {
     property bool paste: false;
     property bool isColonPressed: false;
 
+    Connections {
+        target: chatsModel
+        onChatGroupJoined: {
+            rectangle.visible = true;
```

Comment:
As soon as you set `visible` to `true`, the above conditions do not exist anymore. So if you change to another chat and you're not member yet, you'd still have the Input visible.

Looking at the condition above, it seems like `chatsModel.activeChannel.isMember(profileModel.profile.pubKey)` doesn't update conrrectly. So there is a probably a signal there that we could call instead. maybe making `isMember` a QtProperty and tying a `notify` signal to it that you call instead of `chatGroupJoined` would work.


Also, `rectangle` is a very bad `id` haha. We should change it to either `root` or `chatInputContainer`